src/brs/Burst.java

```diff
@@ -131,10 +131,10 @@ public static void main(String[] args) {
   }
 
   private static void validateVersionNotDev(PropertyService propertyService) {
-    if(VERSION.isPrelease() && !propertyService.getBoolean(Props.DEV_TESTNET)) {
-      logger.error("THIS IS A DEVELOPMENT WALLET, PLEASE DO NOT USE THIS");
-      System.exit(0);
-    }
+//    if(VERSION.isPrelease() && !propertyService.getBoolean(Props.DEV_TESTNET)) {
```

Comment:
will be put back, of course.... just for "dev" purposes. This feature is net agnostic...